tests/functional/report_viewer_api/test_report_filter.py

```diff
@@ -45,44 +45,47 @@ def setUp(self):
         runs = self._cc_client.getRunData(None)
 
         test_runs = [run for run in runs if run.name in run_names]
-
-        self.assertEqual(len(test_runs), 1,
-                         "There should be only one run for this test.")
-        self._runid = test_runs[0].runId
+        self._runids = [r.runId for r in test_runs]
```

Comment:
Don't we want to add at least one testcase where we pass more than one run ids to getRunResults?